osquery/tables/specs/darwin/extended_attributes.table

```diff
@@ -0,0 +1,9 @@
+table_name("extended_attributes")
+description("Get the extended attributes for files (similar to Windows ADS)")
+schema([
+    Column("path", TEXT, required=True),
```

Comment:
Sorry I didn't catch this before, but can you add a third positional argument with the column description?